internal/mbsc/helper.go

```diff
@@ -0,0 +1,30 @@
+package mbsc
+
+import (
+	kmmv1beta1 "github.com/kubernetes-sigs/kernel-module-management/api/v1beta1"
+	v1 "k8s.io/api/core/v1"
+)
+
+//go:generate mockgen -source=helper.go -package=mbsc -destination=mock_helper.go
+
+type Helper interface {
+	SetModuleImageSpec(mbscObj *kmmv1beta1.ModuleBuildSignConfig, imageSpec *kmmv1beta1.ModuleImageSpec, repoSecret *v1.LocalObjectReference)
+}
+
+type helper struct {
+}
+
+func NewHelper() Helper {
+	return &helper{}
+}
+
+func (h *helper) SetModuleImageSpec(mbscObj *kmmv1beta1.ModuleBuildSignConfig, micImageSpec *kmmv1beta1.ModuleImageSpec, repoSecret *v1.LocalObjectReference) {
+	mbscObj.Spec.ImageRepoSecret = repoSecret
```

Comment:
Are we going to update the `imageRepoSecret` on each image update? The secret is generic while the specific image isn't.

Should we get a list of `ModuleImageSpec` instead of a single `ModuleImageSpec` to this method?